api/v1alpha1/hardwareclassification_types.go

```diff
@@ -22,41 +22,43 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-// HardwareClassificationControllerSpec defines the desired state of HardwareClassificationController
-type HardwareClassificationControllerSpec struct {
+// HardwareClassificationSpec defines the desired state of HardwareClassification
+type HardwareClassificationSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
-	// Foo is an example field of HardwareClassificationController. Edit HardwareClassificationController_types.go to remove/update
+	// Foo is an example field of HardwareClassification. Edit HardwareClassification_types.go to remove/update
 	Foo string `json:"foo,omitempty"`
 }
 
-// HardwareClassificationControllerStatus defines the observed state of HardwareClassificationController
-type HardwareClassificationControllerStatus struct {
+// HardwareClassificationStatus defines the observed state of HardwareClassification
+type HardwareClassificationStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 }
 
 // +kubebuilder:object:root=true
+// +kubebuilder:resource:shortName=hwcc;hcc
```

Comment:
What are the two Cs standing for ? hwc and hc might make more sense since you removed controller for the CRD name

Reply:
Agree. Will change it.